Test failures

=================================== FAILURES ===================================
128
___________________ test_portfolio_item_post_with_exception ____________________
129

130  api_request = <function api_request.<locals>.rf at 0x7f903ee50f70>
131

132 @pytest.mark.django_db
133 def test_portfolio_item_post_with_exception(api_request):
134 """Create a new portfolio item for a portfolio"""
135 portfolio = PortfolioFactory()
136 data = {
137   "portfolio": portfolio.id,
138  }
139
    
140 response = api_request("post", "portfolioitem-list", data=data)
141
    
142 assert response.status_code == 400
143 >       assert "Failed to get service offering" in response.data["detail"]
144 E       KeyError: 'detail'
145

146 ansible_catalog/main/catalog/tests/functional/test_portfolio_item_end_points.py:103: KeyError

--- a/ansible_catalog/main/catalog/tests/functional/test_portfolio_item_end_points.py
+++ b/ansible_catalog/main/catalog/tests/functional/test_portfolio_item_end_points.py
@@ -100,7 +100,7 @@ def test_portfolio_item_post_with_exception(api_request):
     response = api_request("post", "portfolioitem-list", data=data)
 
     assert response.status_code == 400
-    assert "Failed to get service offering" in response.data["detail"]
+    assert "service_offering_ref" in response.data["errors"]
 
 
 @pytest.mark.django_db

